articles/dns/dns-troubleshoot.md

```diff
@@ -160,6 +160,24 @@ Example SRV record names (service name 'sip', protocol 'tcp'):
 * [Create DNS record sets and records by using the Azure portal](./dns-getstarted-portal.md)
 * [SRV record type (Wikipedia)](https://en.wikipedia.org/wiki/SRV_record)
 
+## I cannot resolve records in Private DNS Zones using Private Resolver from point to site clients.
+
+**Scenario**:
+
+If you are using Azure provided(168.63.129.16) DNS server on VNET then Point to Site clients will not be able to resolve records Private DNS Zones(including private endpoints) because 168.63.129.16 is not resolvable from on-prem.
+
+![image](https://user-images.githubusercontent.com/101088482/189837694-7343d311-355b-4780-a1ad-407a761e61fc.png)
+
+The following step will help you resolve records from Private DNS zone:
+
+Configuring custom DNS servers on VNET with Private resolver inbound IP address will help you resolve records in private DNS zone (including those created from Private Endpoints). Please note the Private DNS zones must be associated with the VNET that has the Private Resolver.
+
+![Screenshot 2022-09-16 174123](https://user-images.githubusercontent.com/101088482/190636070-3f577f86-7b44-4c3b-97ba-9276d7a06877.jpg)
+
+**More information**:
+
+Private resolver inbound IP address will be pushed to point to site clients as DNS server when VNET is configured with custom DNS.
```

Comment:
we should clarify that it  is the IP addresses that are configured on vnet that are pushed out and hence configuring private resolver endpoints as servers on vnet will push these out to clients automatically and customers can seamlessly resolve from private zones.